pcdsdevices/epics_motor.py

```diff
@@ -89,12 +95,20 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
     EpicsMotor.low_limit_travel.kind = Kind.config
     EpicsMotor.direction_of_travel.kind = Kind.normal
 
+    # Registry of log filters by device name
+    _motion_error_log_filters: dict[str, filter] = {}
```

Comment:
```suggestion
    _motion_error_log_filters: typing.ClassVar[dict[str, typing.Callable]] = {}
```

Maybe? Lazy callable annotation above

Reply:
You're correct, I used the wrong type hinting here.